RELEASE_NOTES.md

```diff
@@ -6,11 +6,18 @@
 
 ## Upgrading
 
-<!-- Here goes notes on how to upgrade from previous versions, including deprecations and what they should be replaced with -->
+- The `ConfigManagingActor` now takes multiple configuration files as input, and the argument was renamed from `config_file` to `config_files`. If you are using this actor, please update your code. For example:
+
+   ```python
+   # Old
+   actor = ConfigManagingActor(config_file="config.yaml")
+   # New
+   actor = ConfigManagingActor(config_files=["config.yaml"])
```

Comment:
Couldn't we support both parameters for update-convenience?

Reply:
Of course we could, but given that we are pre-1.0.0 final I'm not sure if adding the extra maintenance burden is worth it. People will have to update eventually (I don't think it makes sense to keep both options for ever). Specially since we are moving towards using this only in one app, and also eventually I would like the config managing actor to be initialized automatically by the SDK (much like the `microgrid` is, or how `dispatch` also works, with hiding the actor behind a high level interface). Also is a very small and trivial change for apps to update.